docs/examples.md

```diff
@@ -33,6 +70,14 @@ template using the following command line:
 dotnet new gcf-http
 ```
 
+Sample deployment:
+
+```sh
+gcloud functions deploy simple-http --runtime dotnet3 \
```

Comment:
Nit: Separate `--runtime dotnet3` from the command with a newline like with other flags.

See Java for example:
https://cloud.google.com/functions/docs/tutorials/slack#deploying_the_function

Reply:
Done, although it's not clear to me why the name of the function (which is another flag, effectively- just an unnamed one) should stay on the first line.
Personally I preferred it as it was, but I'm not going to argue over it.